File: youqugou-web-manager/src/main/webapp/js/service/seckillAuditService.js
/**
 * 贾运通--2018/12/31
 */
app.service("seckillAuditService", function ($http) {
    this.searchSecKillAudit = function (currentPage, itemsPerPage, searchEntity) {
        return $http.post("../seckillAudit/searchSecKillAudit.do?pageNum=" + currentPage
            + "&pageSize=" + itemsPerPage, searchEntity);
    }

    //状态修改【审核】
    this.updateStatus = function (ids,status) {
        return $http.get('../seckillAudit/updateStatus.do?ids='+ids+"&status="+status);
    }


});

File: youqugou-interface/src/main/java/cn/itcast/core/service/SeckillAuditService.java
package cn.itcast.core.service;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillGoods;

public interface SeckillAuditService {
    PageResult searchSeckillAudit(Integer pageNum, Integer pageSize, SeckillGoods seckillGoods);

    void updateStatus(Long[] ids, String status);
}
